src/fu-util-common.c

```diff
@@ -1263,8 +1263,10 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 	if (tmp != NULL) {
 		g_autofree gchar *desc = NULL;
 		desc = fu_util_convert_description (tmp, NULL);
-		/* TRANSLATORS: multiline description of device */
-		fu_common_string_append_kv (str, idt + 1, _("Description"), desc);
+		if (desc == NULL) {
+			/* TRANSLATORS: multiline description of device */
+			fu_common_string_append_kv (str, idt + 1, _("Description"), desc);
```

Comment:
Shouldn't this be tmp?

Reply:
Gah, that was all sorts of wrong.  Fixed, thanks.